TsRandomizer/LevelObjects/LevelObject.cs

```diff
@@ -131,13 +131,17 @@ public static void AwardFirstFrameItem(Dictionary<int, Item> itemDictionary, Pro
 			if (newItems.Any())
 				GenerateShadowObjects(itemLocations, newItems, seedOptions);
 
-			if (roomChanged || newItems.Any()) AwardFirstFrameItem(itemsDictionary, level.MainHero);
+			var lunais = level.MainHero;
+			if (roomChanged || newItems.Any()) AwardFirstFrameItem(itemsDictionary, lunais);
 
 			KnownItemIds.Clear();
 			KnownItemIds.AddRange(currentItemIds);
 
 			foreach (var obj in Objects)
 				obj.OnUpdate(gameplayScreen);
+
+			if (lunais.AsDynamic()._isHittingHeadOnCeiling && lunais.CurrentState.HasFlag(EAFSM.Skydashing) && lunais.Velocity.Y == 0)
```

Comment:
plz change lunais.CurrentState.HasFlag(EAFSM.Skydashing) to lunais.CurrentState == EAFSM.Skydashing as EAFSM is not a bitflag enum